frontend/package.json

```diff
@@ -100,14 +100,13 @@
     "vue-plausible": "^1.3.2"
   },
   "devDependencies": {
-    "@babel/core": "^7.20.12",
-    "@babel/parser": "^7.20.13",
-    "@babel/preset-env": "^7.20.2",
-    "@babel/preset-typescript": "^7.21.0",
-    "@babel/runtime-corejs3": "^7.21.0",
```

Comment:
This dependency didn't need to be explicitly defined. We do not directly reference it anywhere and it is included anyway as a transient dependency of `preset-env`.